### PR TITLE
Remove package-lock.json

### PR DESCRIPTION
**Pull request by rodion-arr**
Following https://github.com/pkgjs/wiby/pull/38#pullrequestreview-483336058 and https://github.com/pkgjs/wiby/pull/23#issuecomment-667562987, this PR:

- removes `package-lock.json` file
- setups npm to not create it
- adds lock files to `.gitignore`

### CODE REVIEW

**Review by BethGriggs — APPROVED** [review]

**Review by ljharb — APPROVED** [review]
We should also add `package-lock.json` (and the other lockfile filenames) to `.gitignore`.

File: .gitignore [hunk 1]
```diff
@@ -114,3 +114,7 @@ dist
 .yarn/build-state.yml
 .yarn/install-state.gz
 .pnp.*
+
+# lock files
+yarn.lock
+package-lock.json
```
Comment (ljharb) [inline .gitignore:120]:
```suggestion
# Only apps should have lockfiles
npm-shrinkwrap.json
package-lock.json
yarn.lock
```
(the comment is optional but that's what i usually use)

**Comment by dominykas** [comment]
Gah, this has a conflict now - mind updating with master and then I can merge?

**Comment by rodion-arr** [comment]
@dominykas sure, rebased

**Review by dominykas — APPROVED** [review]

**Pull request merged**